bigbluebutton-html5/imports/ui/components/button/base/component.jsx

```diff
@@ -43,9 +90,37 @@ export default class ButtonBase extends Component {
     const remainingProps = Object.assign({}, this.props);
     delete remainingProps.label;
     delete remainingProps.tagName;
+    delete remainingProps.disabled;
+
+    // Delete Mouse event handlers
+    delete remainingProps.onClick;
+    delete remainingProps.onDoubleClick;
+    delete remainingProps.onMouseDown;
+    delete remainingProps.onMouseUp;
+
+    // Delete Keyboard event handlers
+    delete remainingProps.onKeyPress;
+    delete remainingProps.onKeyDown;
+    delete remainingProps.onKeyUp;
 
     return (
-      <Component aria-label={this.props.label} {...remainingProps}>
+      <Component
+        aria-label={this.props.label}
+        aria-disabled={this.props.disabled}
+
+        // Render Mouse event handlers
+        onClick={this.internalClickHandler}
+        onDoubleClick={this.internalDoubleClickHandler}
+        onMouseDown={this.internalMouseDownHandler}
+        onMouseUp={this.internalMouseUpHandler}
+
+        // Render Keyboard event handlers
+        onKeyPress={this.internalKeyPressHandler}
+        onKeyDown={this.internalKeyDownHandler}
+        onKeyUp={this.internalKeyUpHandler}
```

Comment:
/**
 * Event handlers below are used to intercept a parent event handler from 
 * firing when the Button is disabled. 
 * Key press event handlers intercept firing for keyboard users
 */

Is this a clear explanation? I'm open to any suggestions for improvement :)